### DISCUSSION TITLE
"Hide in Subscription Feed" Option for Custom Channels

### DISCUSSION THREAD

**Discussion by dAtA-TRoN** [libre-tube/LibreTube#4411] [Ideas]
Introducing a "Hide in Subscription Feed" option for custom channels would enable users to follow specific channels while filtering out updates that might not align with their current preferences. For instance, let's consider this scenario: I'm a fan of true crime and follow several such channels on YouTube. However, there are times when I'm not in the mood for this type of content, and I'd prefer not to see those thumbnails in my subscription feed. This hide feature would help maintain a subscription feed that caters to individual tastes.

**Comment by luckkmaxx** [comment]
Thats why Groups exists. You can group your channels as you want, and you will see the groups at top of feed tab, below the filters/order line. You can use text or emojis to title your groups. Its nice.

**Reply by dAtA-TRoN** [reply]
It seems there might have been a misunderstanding. My intention is to have the option to hide subscribed channels from the main feed, while still maintaining the ability to access them through the channels. This would provide the opportunity to create many, at times even more unique channels, without cluttering the main feed with lower-quality content.

**Comment by luckkmaxx** [comment]
You can create "TrueCrime" group and anotger one for the rest of channels. The only inconvenience here is that you have to select your prefered group once you are in feed tab. Because "All" group is set by default.

**Reply by dAtA-TRoN** [reply]
Wouldn't it be better to completely eliminate the Main Feed? Instead, users could have the option to choose a Default Channel for themselves. When subscribing to a new YouTube channel, its behavior should resemble that of videos and playlists. After tapping the subscribe button, users should be able to select the specific channels where the content from this subscription would appear. This approach seems like the ideal solution to the problem.
